lib/OrdinaryDiffEqBDF/src/algorithms.jl

```diff
@@ -113,14 +113,17 @@ struct ABDF2{CS, AD, F, F2, P, FDT, ST, CJ, K, T, StepLimiter} <:
     controller::Symbol
     step_limiter!::StepLimiter
 end
-function ABDF2(; chunk_size = Val{0}(), autodiff = true, standardtag = Val{true}(),
+function ABDF2(; chunk_size = Val{0}(), autodiff = AutoForwardDiff(), standardtag = Val{true}(),
         concrete_jac = nothing, diff_type = Val{:forward},
         κ = nothing, tol = nothing, linsolve = nothing, precs = DEFAULT_PRECS,
         nlsolve = NLNewton(),
         smooth_est = true, extrapolant = :linear,
         controller = :Standard, step_limiter! = trivial_limiter!)
+
+    AD_choice = _process_AD_choice(autodiff, chunk_size, diff_type)
```

Comment:
what about standardtag?

Reply:
We can't do the tags in the creation of the alg, since they use the eltype of `u`. They need to be done where that's accessible. For the full DI integration I'm moving the tag creation to `prepare_alg` for the implicit algs, but that's not needed for this.